env/io_posix.h

```diff
@@ -391,12 +391,18 @@ struct PosixMemoryMappedFileBuffer : public MemoryMappedFileBuffer {
 
 class PosixDirectory : public FSDirectory {
  public:
-  explicit PosixDirectory(int fd) : fd_(fd) {}
+  explicit PosixDirectory(int fd, bool is_btrfs)
+      : fd_(fd), is_btrfs_(is_btrfs) {}
   ~PosixDirectory();
   virtual IOStatus Fsync(const IOOptions& opts, IODebugContext* dbg) override;
 
+  virtual IOStatus FsyncForRename(const std::string& filename,
```

Comment:
I guess the fsync after current file renaming is not just for current file but for all previous new files too, so maybe the naming of the function can be more accurate for non-BTRFS file systems. Something like FsyncAfterFileRename() or something like that.

Reply:
make sense, updated.